### DISCUSSION TITLE
how to create a new column in created table in serverpod project

### DISCUSSION THREAD

**Discussion by Shubhamdsingh** [serverpod/serverpod#438] [Q&A] [answered]
@yahu1031

**Comment by yahu1031** [comment, marked as answer]
@Shubhamdsingh When you create models and generate them, you will be getting a pgsql script. connect to DB and run the script and it generates table. Incase if you want to add a column to and existing table with data, then go with a pgsql query in the db.

```sql
ALTER TABLE <table_name> ADD COLUMN <column_name> <data type>;
```

**Comment by vlidholt** [comment]
It can also be useful to use a visual tool to work with the database. I use Postico, which is pretty nice and makes adding columns etc super easy.
